### PR TITLE
Add checklist for things to do when we get a new steering council member

### PR DESCRIPTION
**Pull request by jasongrout**

### CODE REVIEW

**Comment by rgbkrk** [comment]
Quality steps, thanks @jasongrout.

File: people.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
 
-Benevolent Dictator for Life
-============================
+## Benevolent Dictator for Life
 
 Fernando Perez is the Benevolent Dictator for Life (BDFL).
 
-Steering Council
-================
+## Steering Council
```
Comment (willingc) [inline people.md]: Probably should fix the typo in Matthias' last name in this section.

**Comment by willingc** [comment]
@jasongrout Looks good. Would you mind fixing the typo and I will happily push the green button. Yay, process checklist!

**Comment by takluyver** [comment]
> Yay, process checklist!

One of the few meaningful three word sentences that has never been said before ;-)

**Comment by damianavila** [comment]
:+1:

File: people.md (hunk 1 above)
Comment (jasongrout) [inline people.md]: Let's do that on a different PR: https://github.com/jupyter/governance/pull/18

**Comment by willingc** [comment]
I'm going to go ahead and merge this. If you were waiting for the steering council to all weigh in, please feel free to revert. It looks good to me. And for @takluyver: Yay! Yay! Yay! I :heart: process checklists ;-)

Thanks @jasongrout
